system_tests/monitoring.py

```diff
@@ -181,6 +183,53 @@ def test_create_and_delete_metric_descriptor(self):
 
         retry_500(descriptor.create)()
         retry_404_500(descriptor.delete)()
+        descriptor.create()
+        retry_404(descriptor.delete)()
+
+        with self.assertRaises(NotFound):
+            descriptor.delete()
+
+    def test_write_point(self):
+        METRIC_TYPE = ('custom.googleapis.com/tmp/system_test_example' +
+                       unique_resource_id())
+        METRIC_KIND = monitoring.MetricKind.GAUGE
+        VALUE_TYPE = monitoring.ValueType.DOUBLE
+        DESCRIPTION = 'System test example -- DELETE ME!'
+        VALUE = 3.14
+
+        client = monitoring.Client()
+        descriptor = client.metric_descriptor(
+            METRIC_TYPE,
+            metric_kind=METRIC_KIND,
+            value_type=VALUE_TYPE,
+            description=DESCRIPTION,
+        )
+
+        descriptor.create()
+        retry_404(descriptor._fetch)(client, METRIC_TYPE)
```

Comment:
Retrying what?

There's no point to calling GetMonitoredResource here. It tells you nothing about whether the time series datastore knows about the custom metric.

The underlying big picture here is that you normally only create a descriptor for a monitored resource once. In production, you wouldn't care if the first call or two to CreateTimeSeries fails. Experiments, code samples, and system tests are different from production code in that respect.